syntax = "proto3";

package tt;

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
  string identity = 2;
}

// The response message containing the user's name.
message HelloReply {
  string name = 1;
  string identity = 2;
}
